CHANGELOG.md

```diff
@@ -2,6 +2,51 @@
 Important changes to Oh My Fish are recorded here for each release.
 
 
+## [7] - 2020-04-10
+
+Another release that took long but is finally out! v7 support Fish 3.0 and 3.1 and brings a few improvements and bug fixes.
```

Comment:
Small grammar suggestion.

```suggestion
Another release that took long but is finally out! v7 supports Fish 3.0 and 3.1 and brings a few improvements and bug fixes.
```

Reply:
Thanks for reviewing and spotting that one! Fixed 🙌